components/SearchBar/index.tsx

```diff
@@ -20,11 +20,14 @@ import { updateResolvedInfo } from '../../src/store/resolvedInfo'
 import { updatePrev } from '../../src/store/router'
 import smartContract from '../../src/utils/smartContract'
 import toastTheme from '../../src/hooks/toastTheme'
+import useArConnect from '../../src/hooks/useArConnect'
 
 function Component() {
     const zcrypto = tyron.Util.default.Zcrypto()
     const Router = useRouter()
+    const { connect } = useArConnect()
```

Comment:
is this regarding https://www.notion.so/ssiprotocol/Feedback-4-1846f6d4397848c189828e6d0e7f3d2f#95aaa73a396c41e0b94828879def9094 @devwistkey ?

Reply:
Yes, you're right @tralkan